0.118.0 (#15677)
home-assistant/home-assistant.github.io: source/_posts/2020-11-18-release-118.markdown

- [Native types support for templates](#native-types-support-for-templates)
- [Logbook Card](#logbook-card)
- [Header & Footer Editor](#header--footer-editor)
- [Supervisor Network Configuration](#supervisor-network-configuration)
- [Home Assistant OS](#home-assistant-os)
- [Other noteworthy changes](#other-noteworthy-changes)
- [New Integrations](#new-integrations)
- [New Platforms](#new-platforms)
- [Integrations now available to set up from the UI](#integrations-now-available-to-set-up-from-the-ui)
- [If you need help...](#if-you-need-help)
- [Breaking Changes](#breaking-changes)
- [Farewell to the following](#farewell-to-the-following)
- [All changes](#all-changes)

## Home Assistant Conference

Exciting news! Home Assistant is hosting an [online conference](/conference)
this year!

An event to celebrate our community, share ideas, history, creations and
celebrate major milestones! The event will take place on Sunday, December 13.

[Paulus Schoutsen][@balloob], the founder of Home Assistant, will give the
opening and closing keynote, aided by other Home Assistant developers.
In between, we have three different tracks of talks (for everyone, advanced

> users and developers) with a total of 16 different talks and of speakers.

Reviewer: ```suggestion
users and developers) with a total of 16 different talks.
```

Check out the [conference schedule](https://hopin.to/events/home-assistant-conference#schedule)

> to see the subjects of the talks and the list speakers.

Reviewer: ```suggestion
to see the subjects of the talks and the list of speakers.
```

getting a ticket!

For the latest information, check our dedicated [Home Assistant Conference](/conference) page.

<a href="/conference"><img src='/images/conference/conference-header.png' alt='Home Assistant Conference header' class='no-shadow'></a>

## Grid Card

Ever since the start of Lovelace, users have been making grids. However, this was
not easy as it involved fiddling by combining horizontal and vertical stacks —
a confusing and frustrating experience.

[@balloob] to the rescue! This release adds a new Grid Card. You won't have to
write a complicated YAML soup combining both horizontal and vertical stacks.

The Grid Card offers straight forward options to mimic these setups and adds
an option to force each card to be square to boot.

<p class='img'>
<img src='/images/blog/2020-11-0.118/grid-card.png' alt='Screenshot of the Grid Card'>
Screenshot of Grid Card.
</p>

*_Note that the square option won't work with all cards._

## Quick Bar navigation

The Quickbar, which was introduced in Home Assistant 0.117, really took off! We
are happy to see you like it.

If you haven't tried the new Quick Bar yet... you should! It is a quick and

> easy way to get to entities or run command commands. From anywhere in Home

Reviewer: ```suggestion
easy way to get to entities or run commands. From anywhere in Home
```

If so, you probably would have learned, that it is not that simple. In Home
Assistant, the result of a template always has been a piece of text (a string),
even if you made a list.

{% raw %}

```yaml
script:
  my_script:
    alias: Example
    description: Example script with native lists in templates
    variables:
      entities:
        - light.living_room_window
        - light.living_room_table
      color: [255, 0, 0]
    sequence:
      service: light.turn_on
      data:
        entity_id: "{{ entities }}"
        rgb_color: "{{ color }}"
```

{% endraw %}

This is an extremely powerful change to our template engine, that allows for
more advanced future additions and can significantly reduce the complexity of
existing templates in your set up.

It should be mostly compatible with your existing templates; However, be sure
to check the breaking changes section for known possible breaking scenarios.

## Logbook Card

Another new Lovelace card this release. [@zsarnett] added the Logbook Card.

This card allows you to show the logs of one (or more) entities on a Lovelace
Dashboard. This can be really helpful if you want to monitor, for example, the

> motion events  of a sensor or on a camera.

Reviewer: ```suggestion
motion events of a sensor or a camera.
```

<p class='img'>
<img src='/images/blog/2020-11-0.118/logbook-card.png' alt='Screenshot of the new Logbook Card'>
Screenshot of the new Logbook Card.
</p>

[@zsarnett]: https://github.com/zsarnett

## Header & Footer Editor

Entity cards in Lovelace have support for customizing the header and footer of
that card. This is really useful for adding a nice image, a couple of buttons
or a graph to the card.

Up until now, you needed some YAML magic to achieve that. Thanks to
[@zsarnett] we now have a Header & Footer editor available in the UI making
this feature much easier to use.

<p class='img'>
<img src='/images/blog/2020-11-0.118/header-footer-editor.png' alt='Screenshot of the new Header/Footer editor.'>
Screenshot of the new Header/Footer editor.
</p>

[@zsarnett]: https://github.com/zsarnett

## Supervisor Network Configuration

> Excepted to be available soon, a new Supervisor version which includes a new

Reviewer: ```suggestion
Expected to be available soon, a new Supervisor version which includes a new
```

network layer with added support for wireless networks, multiple network
interfaces and even VLANs.

> This should cover most of the future request we had around networking.

Reviewer: ```suggestion
This should cover most of the feature requests we had around networking.
```

<p class='img'>
<img src='/images/blog/2020-11-0.118/supervisor-network.png' alt='New Supervisor Network configuration UI'>
New Supervisor Network configuration UI
</p>

The next step is that we use all the information from our new network backend

> to simplify some Add-ons. Also, updates for Snapshot/Restore these network

Reviewer: ```suggestion
to simplify some add-ons. Also, updates for Snapshot/Restore of these network
```

settings are planned.

## Home Assistant OS

The last few days, two new Home Assistant OS releases have been published.

While 4.17 continues to make small improvements in the 4.x release series, the
latest pre-release 5.5 comes closer to our goals to declare 5.x release series
stable. In particular, this pre-release includes the Linux 5.4 kernel for all
Raspberry Pis! We also intend to declare 64-bit the recommended installation
method for Raspberry Pi 4.

Release 5.5 is also the first release to support Hardkernel's ODROID-C4,
a very cost-effective single board computer featuring the Amlogic S905X3 SoC
and 4GB of DDR4 memory (thanks @ubergeek801). Testers welcome!

## Other noteworthy changes

- The new Nest SDM integration now supports thermostats! Thanks, [@allenporter]!
- Entity pickers in the UI will now show the full name of the entities.

>   Thank [@spacegaier]

Reviewer: ```suggestion
  Thanks, [@spacegaier]!
```

- The Roon media player now supports media browsing but also has support
  for grouping and transferring between different players. Thanks [@pavoni]
- [@cnorick] added a button to duplicate Scripts from the UI.
- The Shelly integration now has real-time status updates, which has been
  implemented by [@thecode]!
- [@apop880] prepared the WLED integration for Christmas by adding a service
  for loading WLED presets.
- Date pickers in entity cards have been replaced with a modern, good looking
  version. Thanks, [@thomasloven]!

> - [@joshmcrty] Made numbers shown in the frontend are now being formatted in a

Reviewer: ```suggestion
- [@joshmcrty] Made numbers shown in the frontend formatted in a
```

  way that matches your language settings.

> - [@spacegaier] has been busy improving the frontend more accessible to people

Reviewer: ```suggestion
- [@spacegaier] has been busy improving the frontend, making it more accessible to people
```

  with disabilities. In this case, ARIA (Accessible Rich Internet Applications)
  labels have been added, so screen readers can use our `+` buttons. Thanks!

[@cnorick]: https://github.com/cnorick
[@joshmcrty]: https://github.com/joshmcrty
[@spacegaier]: https://github.com/spacegaier
[@thomasloven]: https://github.com/thomasloven

## New Integrations

We welcome the following new integration this release:

- [Color Extractor][color_extractor docs], added by [@GenericStudent]

## New Platforms

The following integration got support for a new platform:

> - [1-Wire][onewire docs] now has supports binary sensors and switches,

Reviewer: ```suggestion
- [1-Wire][onewire docs] now supports binary sensors and switches,
```

  added by [@epenet]

## Integrations now available to set up from the UI

The following integrations are now available via the Home Assistant UI:

- [1-Wire][onewire docs], done by [@epenet]
- [Epson][epson docs], done by [@pszafer]

## If you need help...

...don't hesitate to use our very active [forums](https://community.home-assistant.io/) or join us for a little [chat](https://discord.gg/c5DvZ4e).

Experiencing issues introduced by this release? Please report them in our [issue tracker](https://github.com/home-assistant/core/issues). Make sure to fill in all fields of the issue template.

<!--more-->

## Breaking Changes

Below is a listing of the breaking change for this release, per subject or
integration. Click on one of those to read more about the breaking change
for that specific item.

<details>
  <summary><b>PTVSD</b></summary>
  <p>

> The `ptvsd` integration is been marked as deprecated and will be removed in

Reviewer: ```suggestion
The `ptvsd` integration has been marked as deprecated and will be removed in
```
